modules/ui/commit_warnings.js

```diff
@@ -92,12 +97,72 @@ export function uiCommitWarnings(context) {
 
             items.selectAll('.issue-message')
                 .text('')
-                .each(function(d) {
+                .each(function(d){
                     return d.message(context)(d3_select(this));
-                });
+                 });
+
+            // Adding ellipsis if there are more issues
+            if (hasMoreIssues) {
+                // IIFE to create a new scope for each instance
+                (function(localContainer) {
+
+                    var ellipsisItem = localContainer.select('ul').selectAll('.ellipsis')
+                        .data([0]);
+
+                    ellipsisItem.enter()
+                        .append('div')
+                        .attr('class', 'ellipsis')
+                        .append('button')
+                        .text('...More warnings')
```

Comment:
Labels should be translatable. See https://github.com/openstreetmap/iD/wiki/How-to-Use-iD%27s-Localizer-Module for a tutorial for how the localizer module is used.

Reply:
yup! i think that dropdown kind of UI looks better I'll try to fix that and also try to fix those unrelated changes

Reply:
![image](https://github.com/user-attachments/assets/7ed9a5f9-9f65-4084-b2a7-6b5d2c7c468d)
I've made the changes which you told me to